File: java/src/main/java/mediator/Request/OrderRequest.java
package mediator.Request;

import model.Order;

public class OrderRequest extends Request{
    private Order order;

    public OrderRequest(String service, String type) {
        super(service, type);
    }

    public Order getOrder() {
        return order;
    }

    public void setOrder(Order order) {
        this.order = order;
    }
}

File: csharp/SEP3WebAPI/Mediator/Requests/OrderRequest.cs
using System.Collections;
using System.Collections.Generic;
using SEP3Library.Models;

namespace SEP3WebAPI.Mediator.Requests {
    public class OrderRequest : Request {
        public int Index { get; set; }
        public Order Order { get; set; }
        public IList<Order> Orders { get; set; }
    }
}